R/diagnose_design.R

```diff
@@ -220,6 +220,7 @@ calculate_diagnosands <- function(simulations_df, diagnosands, group_by_set) {
 
 calculate_diagnosands_single_design <- function(simulations_df, diagnosands, group_by_set) {
   group_by_list <- simulations_df[, group_by_set, drop = FALSE]
+  group_by_list[is.na(group_by_list)] <- "NA"
```

Comment:
Are we sure that we aren't accidentally changing numbers into text bc of this?

Reply:
For pretty printing you might want `""` instead of `"NA"`.

Reply:
ooh, i'll check about numbers and text

Reply:
add_grouping_variables could contain columns with numbers.